client/src/components/ratings/ratings.jsx

```diff
@@ -0,0 +1,47 @@
+const React = require('react');
+
+const Ratings = (props) => {
+  return (
+    <div>
+    Ratings
+      <div>
+      Accuracy
+        {props.data.length !== 0
+          && Math.round((props.data.map(data => data.accuracy)
```

Comment:
is there a way to pull this out as a function that you have above the return statement? so something like

`const getAverage = (reviewField) => {
  Math.round((props.data.map(data => data.reviewField)
    .reduce((acc, cur) => acc + cur)) / props.data.length)}
}`

and then call it in each field: 
`{props.data.length !== 0
  && getAverage('accuracy')}`

Reply:
good idea! ill refactor1